lib/src/webcrypto/webcrypto.hmac.dart

```diff
@@ -25,7 +25,44 @@ part of 'webcrypto.dart';
 ///  * [JWK] format using [HmacSecretKey.importJsonWebKey].
 ///
 /// A random key can also be generated using [HmacSecretKey.generateKey].
-///
+/// 
+/// **Example**
+/// ```dart
+/// import 'package:webcrypto/webcrypto.dart';
+/// import 'dart:convert';
+/// 
+/// Future<void> main() async {
+///   // Generate a new key using SHA-256 and an optional length parameter.
```

Comment:
I can't parse this comment.

The length parameter is optional, but that's irrelevant, the comment should probably explain what the value passed is.

Also omitting length is fine, in this case it has no effect.

I suggest that we leave length of the example.

Reply:
Removed the length parameter.
Improved the comment